store/tikv/safepoint.go

```diff
@@ -124,7 +124,7 @@ func saveSafePoint(kv SafePointKV, key string, t uint64) error {
 }
 
 func loadSafePoint(kv SafePointKV, key string) (uint64, error) {
```

Comment:
```suggestion
func loadSafePoint(kv SafePointKV) (uint64, error) {
```
how about? And `saveSafePoint` may have the same issue?

Reply:
Good point, I've eliminated the `key` parameter altogether!